AzureIotPnpDps.X/platform/application_manager.c

```diff
@@ -48,61 +52,68 @@ void  wifiConnectionStateChanged(uint8_t status);
 void application_init() {
 	uint8_t mode = WIFI_DEFAULT;
 
+	LED_init();
 	LED_test();
-#if CFG_ENABLE_CLI     
+#if CFG_ENABLE_CLI
 	CLI_init();
 	CLI_setdeviceId(attDeviceID);
 #endif   
 	debug_init(attDeviceID);
-    //debug_setSeverity(SEVERITY_DEBUG);
-    
+	debug_setSeverity(SEVERITY_DEBUG);
+	//debug_setSeverity(SEVERITY_INFO);
+
 	// Initialization of modules where the init needs interrupts to be enabled
 	cryptoauthlib_init();
 	if (cryptoDeviceInitialized == false)
 	{
-		debug_printError("APP: CryptoAuthInit failed");
+		debug_printError("   APP: CryptoAuthInit failed");
 	}
-    
+
 #ifdef HUB_DEVICE_ID
 	attDeviceID = HUB_DEVICE_ID;
-#else    
+#else
 	// Get serial number from the ECC608 chip 
 	retValCryptoClientSerialNumber = CRYPTO_CLIENT_printSerialNumber(attDeviceID_buf);
 	if (retValCryptoClientSerialNumber != ATCA_SUCCESS)
 	{
 		switch (retValCryptoClientSerialNumber)
 		{
 		case ATCA_GEN_FAIL:
-			debug_printError("APP: DeviceID generation failed, unspecified error");
+			debug_printError("   APP: DeviceID generation failed, unspecified error");
 			break;
 		case ATCA_BAD_PARAM:
-			debug_printError("APP: DeviceID generation failed, bad argument");
+			debug_printError("   APP: DeviceID generation failed, bad argument");
+			break;
 		default:
-			debug_printError("APP: DeviceID generation failed");
+			debug_printError("   APP: DeviceID generation failed");
 			break;
 		}
 	}
-    else
-    {
-        // To use Azure provisioning service, attDeviceID should match with the device cert CN,
-        // which is the serial number of ECC608 prefixed with "sn" if you are using the 
-        // the microchip provisioning tool for PIC24.
-        strcat(buf, SN_STRING);
-        strcat(buf, attDeviceID_buf);
-        attDeviceID = buf;
-        debug_print("CRYPTO_CLIENT_printSerialNumber %s", attDeviceID);
-    }
-#endif    
-    
+	else
+	{
+		// To use Azure provisioning service, attDeviceID should match with the device cert CN,
+		// which is the serial number of ECC608 prefixed with "sn" if you are using the 
+		// the microchip provisioning tool for PIC24.
+		strcat(buf, SN_STRING);
+		strcat(buf, attDeviceID_buf);
+		attDeviceID = buf;
+		debug_printGood("   APP: CRYPTO_CLIENT_printSerialNumber %s", attDeviceID);
+	}
+#endif
+
 #if CFG_ENABLE_CLI   
 	CLI_setdeviceId(attDeviceID);
 #endif   
 	debug_setPrefix(attDeviceID);
 
+	mode = WIFI_DEFAULT;
+
 	if (!SW0_Value && SW1_Value)
 	{
-		//SW0 only
-		mode = WIFI_SOFT_AP;
+		// AP Mode is not enabled
+		//SW0 only 
+		//debug_printGood("   APP: Starting WIFI_SOFT_AP");
+		//mode = WIFI_SOFT_AP;
```

Comment:
is this supposed to be commented out?

Reply:
AP mode wasn't working, but I will double check

Reply:
AP Mode was never working but keeping them as comment in case someone would like to add/enable